src/System.Net.HttpListener/tests/HttpListenerWebSocketTests.cs

```diff
@@ -35,7 +35,7 @@ public void Dispose()
         }
 
         [ActiveIssue(22164, TargetFrameworkMonikers.Uap)] // Fails in RS2 only (works in RS3).
-        [ConditionalTheory(nameof(IsNotWindows7))]
+        [ConditionalTheory(nameof(PlatformDetection.ClientWebSocketPartialMessagesSupported), nameof(IsNotWindows7))]
```

Comment:
We don't need the "nameof(IsNotWindows7)" anymore.  By adding the PlatformDetection.ClientWebSocketPartialMessagesSupported item, it already is always "not Windows 7".